content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,214 @@
+
+## Steps 
+
+ - **1 General settings**
+ - **2 Multistore (If multistore is enabled)**
+ - **3 Shipping locations and costs**
+ - **4 Size, weight, and group access**
+ - **5 Summary**
+
+## Fields
+
+ - **Maximum package width (_Dimension unit_)** This is a field. The following tooltip is displayed: "Maximum width managed by this carrier. Set the value to "0", or leave this field blank to ignore. The value must be an integer." The dimension unit mentioned in parenthesis is the one defined in International > Localization > Localization
+If what is filled is not an integer, the following error message is displayed: "The max_width field is invalid."
+ - **Maximum package height (_dimension unit_)** This is a field. The following tooltip is displayed: "Maximum height managed by this carrier. Set the value to "0", or leave this field blank to ignore. The value must be an integer." The dimension unit mentioned in parenthesis is the one defined in International > Localization > Localization
+If what is filled is not an integer, the following error message is displayed:  "The max_height field is invalid."
+ - **Maximum package depth (_dimension unit_)** This is a field. The following tooltip is displayed: "Maximum height managed by this carrier. Set the value to "0", or leave this field blank to ignore. The value must be an integer.". The dimension unit mentioned in parenthesis is the one defined in International > Localization > Localization.
+If what is filled is not an integer, the following error message is displayed: "The max_depth field is invalid."
+ - **Maximum package weight (_Weight unit_)** This is a field. The following tooltip is displayed: "Maximum height managed by this carrier. Set the value to "0", or leave this field blank to ignore.". The weight unit mentioned in parenthesis is the one defined in International > Localization > Localization.
+ - **Group access** The ID and the name of all existing customers' groups are displayed. A checkbox is displayed before each group to select or unselect them. A global checkbox allows selecting/unselecting all customers' groups in one click. By default, all the customers' groups are checked.
+The following tooltip is displayed: "Mark the groups that are allowed access to this carrier."
+
+## Call to action
+
+  - **Previous.** If there is an error, this button is disabled. After clicking on it, the previous step is displayed.
+ - **Next.** If there is an error, this button is disabled. After clicking on it, the next step is displayed.
+ - **Finish.** If there is an error, this button is disabled. After clicking on it, the carrier is saved and the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page is displayed with the following success message: "Successful creation" if the carrier was added or "Successful update" if the carrier was edited.
+
+# Summary (5th step)
```

Comment:
Both call to action logo and steps are described at the begining. Could you please remove them from this section?